app/serializers/discussion_reader_serializer.rb

```diff
@@ -33,10 +33,6 @@ def discussion_reader_id
     object.id
   end
 
-  def discussion_reader_volume
```

Comment:
This gets used in `discussion_model.coffee`, I wonder if we need to account for that?

Reply:
Also, wonder if volume changes are not strictly necessary for group export